### DISCUSSION TITLE
Help Needed: Missing Unedited Photo After Moving Files

### DISCUSSION THREAD

**Discussion by EmmeMax** [deckerst/aves#1184] [Q&A]
Hi everyone,

I hope you're all doing well!

I'm having a bit of an issue and could use some help. I had two versions of the same image: one edited in Lightroom Mobile (Image A) and one unedited (Image B). After moving all the photos from my trip to a folder, the unedited version seems to have disappeared. Now, I’m seeing two edited images in Aves, and only the edited one shows up in Google Photos.

My question is: what happened to the unedited image?

When I moved the pictures, I got a popup asking if I wanted to replace an image already in the album, and I clicked to replace it. But I'm sure the image that got replaced wasn't the unedited one I'm talking about.

So, any idea why I’m seeing two of the same edited images and where the original unedited one might have gone? Is it possible to undo the album/moving images to the album?

Thanks a lot for any insights!

**Comment by deckerst** [comment]
From what you say it sounds like image A and B had the same name, so when moving both to the same folder in Aves, it asked what to do (rename/skip/replace). Replacing means either A or B will be overwritten by the other, depending on the order of the operation.

The overwritten file is not recoverable. It wasn't moved to the bin.

Let's see how things can be improved for the future. For example, when the app prompted you because of the name clash, should it be clearer that "replacing" means files will be overwritten?

Possible improvement 1: would it have helped you pick the right option if the app showed the conflicting names? 

Possible improvement 2: files replaced in that way are currently not moved to the bin. Should they be, just as if the user manually deleted them first?

Bug: the app shouldn't show 2 items when one of them got replaced during the operation.
